### PR TITLE
Add calculator component.

### PR DESCRIPTION
**Pull request by BenjaminMatembe**
**In this feature of the project I:**

- Implemented the design of the UI of the calculator using JSX and CSS
- Used functional components
- Created at least one child component (`CalculatorChild.js`)
- Tested in the browser and UI was working well without problems.

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+          <div className="light_grey center"><p>1</p></div>
+          <div className="light_grey center"><p>2</p></div>
+          <div className="light_grey center"><p>3</p></div>
+          <div className="orange center"><p>+</p></div>
+        </div>
+        <div className="between" />
+        <div className="second">
+          <div className="light_grey center bigger"><p>0</p></div>
+          <div className="light_grey center"><p>.</p></div>
+          {/* <div className="light_grey center"><p></p></div> */}
```
Comment (Davidosky007) [inline src/components/Calculator.js]: * Kindly remove all commented code from your codebase it makes your codebase bulky and unprofessional

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
Hi @BenjaminMatembe 👋 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
## Highlights
* React components used properly 👍 
* Well written PR 👍 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @davidsoky007 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by DammyShittu — APPROVED** [review]
Hi @BenjaminMatembe,

## APPROVED 🟢

## Highlights
✔️ Clean code

Great implementation of all the requirements in this milestone. 👏 

_You have done an amazing job 👍_

Your project is complete!  There is nothing else to say other than... it's time to merge it. :shipit: 

Congratulations! 🎉 

## CHEERS AND HAPPY CODING!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by BenjaminMatembe** [comment]
Hi @Davidosky007 Thank you for the highlights, @DammyShittu Thank you for taking the time to review and approve my PR.

**Pull request merged**